logging/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+find_package(Boost 1.54 REQUIRED COMPONENTS log)
+
+include_directories(${Boost_INCLUDE_DIRS})
+include_directories(${LEATHERMAN_CATCH_INCLUDE})
```

Comment:
I'm going to wrap these includes in a macro that ensure the dependency libraries are enabled, and bail if not.

When we're in the `find_package` world, I'll be sure to enable proper dependency tracking between leatherman components.